bot/seasons/pride/pride_avatar.py

```diff
@@ -123,6 +123,12 @@ def crop_ring(ring, px):
 
         await ctx.send(file=file, embed=embed)
 
+    @prideavatar.command()
+    async def flags(self, ctx):
+        """This lists the flags that can be used with the prideavatar command."""
+        options = "\n".join(set(OPTIONS.values()))
+        await ctx.send(f"**I have the following flags:**\n{options}")
```

Comment:
Instead of a plain text message, I would like to see this in a formatted embed. Maybe you could sort them as well, which may make looking up a name in the list somewhat easier (since it's relatively long).

This is just a boring example, without color, of what I mean (without sorting):

![2019-06-03_21-16-07](https://user-images.githubusercontent.com/33516116/58828107-f4889280-8644-11e9-90e5-72b611e9c50a.png)